libsolidity/codegen/ABIFunctions.cpp

```diff
@@ -357,8 +378,35 @@ string ABIFunctions::conversionFunction(Type const& _from, Type const& _to)
 			break;
 		}
 		case Type::Category::FixedPoint:
-			solUnimplemented("Fixed point types not implemented.");
+		{
+			FixedPointType const& fromFixedType = dynamic_cast<FixedPointType const &>(_from);
+			if (toCategory == Type::Category::Integer)
+			{
+				body =
+					Whiskers("converted := <toInt>(<cleanFixed>(value))")
+					("cleanFixed", cleanupFunction(_from, false))
+					("toInt", scaleFixedFunction(fromFixedType.fractionalDigits(), 0, fromFixedType.isSigned()))
+					.render();
+			}
+			else if (toCategory == Type::Category::FixedPoint)
+			{
+				FixedPointType const &toFixedType = dynamic_cast<FixedPointType const &>(_to);
+				body =
+					Whiskers("converted := <scale>(<cleanFixed>(value))")
+					("scale", scaleFixedFunction(fromFixedType.fractionalDigits(), toFixedType.fractionalDigits(), fromFixedType.isSigned()))
```

Comment:
What if one type is signed and the other isn't?

Reply:
We only care about if the from type is signed, since that is what's going to affect division/multiplication. Once that's done, if the other type is unsigned, it's reinterpreted as unsigned.